app/mailers/amazon_ses_mailer.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# rubocop:disable Rails/ApplicationMailer
+# :nocov:
+class AmazonSesMailer < ActionMailer::Base
+  include MailerAnalyticsEvents
+  helper NotifyViewsHelper
+
+  helper_method :uid, :utm_campaign
+
+  after_action :trigger_dfe_analytics_email_event
+
+  self.delivery_method = :smtp unless Rails.env.test?
+
+  default from: "ats.teachingvacancies@service.education.gov.uk"
+
+  def send_email(to:, subject:)
+    @to = to
+    mail(to: to, subject: subject, delivery_method_options: {
+      user_name: Rails.application.credentials.dig(:smtp, :user_name),
+      password: Rails.application.credentials.dig(:smtp, :password),
```

Comment:
:raising_hand: The service uses AWS Parameter Store to store these secrets ) we have an entry per env on ...`/app/secrets`
I remember having to add the Rails application credentials just because we had to store a massive private key for GovUK One Login that exceeded parameter store limits. But if possible... we should keep all the secrets together.